MAINTAINERS.md

```diff
@@ -4,12 +4,9 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 
 ## Current Maintainers
 
-| Maintainer        | GitHub ID                                         | Affiliation |
-| ----------------- | ------------------------------------------------- | ----------- |
-| Anan              | [ananzh](https://github.com/ananzh)               | Amazon      |
-| Bishoy Boktor     | [boktorbb-amzn](https://github.com/boktorbb-amzn) | Amazon      |
-| Rocky             | [kavilla](https://github.com/kavilla)             | Amazon      |
-| Sean Neumann      | [seanneumann](https://github.com/seanneumann)     | Amazon      |
-| Tommy Markley     | [tmarkley](https://github.com/tmarkley)           | Amazon      |
-| Josh Romero       | [joshuarrrr](https://github.com/joshuarrrr)       | Amazon      |
-| Ashwin P Chandran | [ashwin-pc](https://github.com/ashwin-pc)         | Amazon      |
```

Comment:
from reading the instructions on the issue, I believe any existing maintainers should either remain or be move to an emeritus section: https://github.com/opensearch-project/.github/issues/125

Reply:
This is rebaselining so i think that we can remove anyone who hasnt contributed to it. All the people being removed have not contributed to the repo.